src/common/meta/src/ddl/drop_database.rs

```diff
@@ -105,7 +116,13 @@ impl DropDatabaseProcedure {
                 tables: None,
             },
             state,
-        })
+        };
+        procedure
```

Comment:
Perhaps we should let the framework call `on_recover` (after executing `from_json`).